webapp/ascii_art.py

```diff
@@ -1,22 +1,41 @@
 import os
-
+import time
 from flask import Flask, flash, request, redirect, render_template, url_for
 from PIL import Image
 from werkzeug.utils import secure_filename
-
+import warnings
 from make_art import convert_image_to_ascii
 
-UPLOAD_FOLDER = "./images"
+IMG_FOLDER = os.path.join('static', "IMG")
 ALLOWED_EXTENSIONS = {"png", "jpg", "jpeg"}
-
 app = Flask(__name__)
-app.config["UPLOAD_FOLDER"] = UPLOAD_FOLDER
+app.config['MAX_CONTENT_LENGTH'] = 16 * 1024 * \
+    1024  # File can not be largr than 16 Mb
+app.config["UPLOAD_FOLDER"] = IMG_FOLDER
 
 
 def allowed_file(filename):
     return "." in filename and filename.rsplit(".", 1)[1].lower() in ALLOWED_EXTENSIONS
 
 
+def check_for_folder():
+    '''Basically it will check if there are already existing folder named images and gallery or not'''
+    if not os.getcwd() == "webapp":
```

Comment:
This will always return `True` because `os.getcwd()` returns an absolute path, not the folder name, so we will always get a warning.
An easy way to accomplish what you want is using `pathlib`:

```python
>>> from pathlib import Path
>>> Path.cwd().name
'webapp'
```

Reply:
> Reference
oh thanks, I will edit it out